README.md

```diff
@@ -1 +1,41 @@
-# My-portofolio
\ No newline at end of file
+# My-Portfolio
+
+> Portfolio: setup and mobile version skeleton
+## Built With
+
+- HTML
+- CSS
+
+## Screenshot
+
+![mobile](https://user-images.githubusercontent.com/38283436/143400448-31f5b880-6584-43b0-98c0-f1a9c4f2f34d.PNG)
+
+
+## Getting Started
+
+**The aim of the project is to set up a new repository and prepare it for development using best practices and
+build the first 3 sections of the mobile website using the template provided in Figma**
```

Comment:
**Optional**

- [ ] I believe this should be in the title description. While the `Getting Started` contains instructions on setting up your project locally. It will be nice if you can consider and implements this suggestion. See [link](https://github.com/microverseinc/readme-template) to microverse readme template for a refresher.

Implementing this optional suggestion will not in any way affect the status of your project. However I highly recommend it so as to make your readme more descriptive.

Reply:
Okay. I will change it. Thank you.